src/App.tsx

```diff
@@ -1,57 +1,113 @@
-import React from 'react';
```

Comment:
- [ ] Try to move all logic to another components. So App.tsx is the entry point. This component is overloaded. try to split into smaller components